packages/loader/src/gltf/extensions/EXT_texture_webp.ts

```
@@ -8,25 +8,36 @@ import { GLTFExtensionMode, GLTFExtensionParser } from "./GLTFExtensionParser";
 interface EXTWebPSchema {
   source: number;
 }
+function checkWebpSupport(): Promise<boolean> {
+    return new Promise((resolve) => {
```

ESLint

[error] Delete `··`

(prettier/prettier)